README.md

```diff
@@ -21,7 +21,7 @@ As the current limitation about the number of automated builds on hub.docker.com
 * `stackdriver`, `papertrail`, `syslog` images (x86_64/arm64) won't be published anymore
 * `logentries`, `loggly`, `logzio`, `s3` arm64 images won't be published anymore (x86_64 only supported)
 
-If you want to use above non published images, build it by yourself. Dockefile itself is still maintained in this repository.
+If you want to use above non published images, build it by yourself. Dockerfile itself is still maintained in this repository.
```

Comment:
Could you fix template/README.md.erb instead?
README.md is a generated file from it, so that it will be overwritten on every release.